pyvista/plotting/plotting.py

```diff
@@ -4142,11 +4153,7 @@ def show(self, title=None, window_size=None, interactive=True,
         # If user asked for screenshot, return as numpy array after camera
         # position
         if return_img or screenshot is True:
-            return cpos, self.last_image
-
-        # Return last used camera position unless within a doctest
-        if '_pytest.doctest' not in sys.modules:
-            return cpos
+            return self.last_image
 
```

Comment:
Shouldn't we enable `return_cpos` for `show()` just like for `plot()`? Wouldn't this make it easier for folks who've been relying on the returned value for whatever reason?

Reply:
I'll make this a theme default.